src/main/java/com/sda/controller/CostumeController.java

```diff
@@ -4,10 +4,7 @@
 import com.sda.model.User;
 import com.sda.service.OrderService;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.ModelAttribute;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.*;
```

Comment:
minor:
Lepiej unikac takich importów, bo importujemy całą masę niepotrzebnych klas. Można to zmienic w ustawieniach intellij - https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-